viite-UI/src/view/NodeLayer.js

```diff
@@ -615,7 +581,10 @@
     this.layerStarted = function (eventListener) {
       eventListener.listenTo(eventbus, 'roadLinks:fetched', function () {
         if (applicationModel.getSelectedLayer() === 'node') {
-          redraw();
+          me.clearLayers(layers);
+          if (applicationModel.getCurrentAction() === -1) {
```

Comment:
(TODO, TOTHINK: For this PR, this is just a leftover from refactoring... ...but what does this do? Does this do at least something, some time? Why is this here? Explanation would be nice! :) )